src/components/encoding-pane/property-editor-schema.ts

```diff
@@ -132,12 +262,45 @@ export function getPropertyEditorSchema(prop: string, nestedProp: string, propTa
       type: 'string'
     };
     return generateSchema(prop, nestedProp, DEFAULT_SELECT_UISCHEMA, propTab, schemaProperty);
+  } else if (prop === 'size') {
+    const schemaProperty: StringSchema = {
```

Comment:
Pretty much almost all cases of this method need to type 

```
{
  title, 
  type: string
}
```

I wonder if there is a way to reduce reduntant typing required.